src/service/poc_mobile.proto

```diff
@@ -80,6 +83,43 @@ message subscriber_location_ingest_report_v1 {
   subscriber_location_req_v1 report = 2;
 }
 
+enum signal_level {
+  NO = 0;
+  LOW = 1;
+  MEDIUM = 2;
+  HIGH = 3;
+}
+
+// Radio signal level and power in the h3 hex.
+message radio_hex_signal_level {
+  // H3 hex tile covered by the radio
+  string location = 1;
+  signal_level signal_level = 2;
+  // Signal power of the radio in dBm (decibel-milliwatts)
+  double signal_power = 3;
+}
+
+// Array of radio_hex_signal_level messages
+message signal_levels { repeated radio_hex_signal_level value = 1; }
+
+message coverage_object_req_v1 {
+  bytes uuid = 1;
+  string cbsd_id = 2;
+  uint64 coverage_claim_time = 3;
```

Comment:
Could you add a comment about the measurement of `coverage_claim_time`?
ms or sec?

Reply:
Sure. I'm assuming it's a timestamp in seconds? Or ms?

Reply:
seconds